src/adapters/AxelarAdapter.sol

```diff
@@ -44,13 +46,19 @@ contract AxelarAdapter is Auth, IAxelarAdapter {
     // Administration
```

Comment:
Given it also includes the view method:
```suggestion
    // Network wiring
```

Reply:
Done in [edaa246](https://github.com/centrifuge/protocol/pull/654/commits/edaa2469aabb5d723c442f95435b604da91b86a7)